network/alsp/report.go

```diff
@@ -0,0 +1,79 @@
+package alsp
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/network"
+)
+
+// MisbehaviorReport is a report that is sent to the networking layer to penalize the misbehaving node.
+// A MisbehaviorReport reports the misbehavior of a node on sending a message to the current node that appears valid
+// based on the networking layer but is considered invalid by the current node based on the Flow protocol.
+//
+// A MisbehaviorReport consists of a reason and a penalty. The reason is a string that describes the misbehavior.
+// The penalty is a value that is deducted from the overall penalty of the misbehaving node. The penalty is
```

Comment:
```suggestion
// The penalty is a value that is deducted from the overall score of the misbehaving node. The score is
```
Suggesting to differentiate terminology for penalty (decrease in "score" associated with a misbehaviour) and "score" - the current sum of all previous penalties and decays.